### PR TITLE
chore(quick-brick-user-account-ui-component): add screen resolver to user account component

### PR DESCRIPTION
**Pull request by kononenkoAnton**
## Description
Add to user account component a logic to parse screen with context that can be defined in plugin styles
context example. `login_{{ctx/countryCode}}`

[Monday ticket](https://applicaster.monday.com/boards/1615228456/views/35072856/pulses/3490521588?userId=16469543)

Screenshot 2022-11-28 at 4 26 54 PM

<details>
  <summary>Platforms Affected</summary>
  
- [ ] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [ ] iOS
- [ ] LG TV (WebOS)
- [ ] Samsung TV (Tizen)
- [ ] Web
</details>

## Known or Pending Issues

`fill in this section with potential issues / limitations the reviewer(s) should know about`

## Screenshots

`If there is some kind of ui change please share screenshots`

## Version Info

`Does this update require a certain version of quickbrick, or does it rely on another pr to work?`

### CODE REVIEW

File: plugins/quick-brick-user-account-ui-component/manifests/styles.config.js [hunk 1]
```diff
@@ -72,9 +73,50 @@ const mobile = [
...
+          },
+        ],
+      },
+      {
+        key: "button_1_login_screen_type_default",
+        type: "text_input",
+        initial_value: "",
+        label: "Login Screen Time Default",
+        tooltip:
+          "Add screen mapping login_{{ctx/countryCode}}in this format, fallback logic",
```
Comment (ranmeirman) [inline plugins/quick-brick-user-account-ui-component/manifests/styles.config.js:112]: @kononenkoAnton I don't understand the tooltip and also you are missing space before the `in`

File: plugins/quick-brick-user-account-ui-component/manifests/styles.config.js [hunk 2]
```diff
@@ -46,6 +46,7 @@ const mobile = [
           { text: "Adobe primetime", value: "adobe_primetime" },
           { text: "LoginFlow", value: "login_flow" },
           { text: "Other", value: "other" },
+          { text: "Login resolver", value: "login_resolver" },
```
Comment (ranmeirman) [inline plugins/quick-brick-user-account-ui-component/manifests/styles.config.js:49]: Please change the label to `Type Mapping Resolver`

File: plugins/quick-brick-user-account-ui-component/manifests/styles.config.js [hunk 3]
```diff
@@ -245,9 +288,50 @@ const mobile = [
...
+            key: "styles/button_1_login_type",
+          },
+        ],
+      },
+      {
+        key: "button_2_login_screen_type",
+        type: "text_input",
+        initial_value: "",
+        label: "Login Screen Time",
+        tooltip: "Add screen mapping login_{{ctx/countryCode}}in this format",
```
Comment (ranmeirman) [inline plugins/quick-brick-user-account-ui-component/manifests/styles.config.js:312]: same comment as above

File: plugins/quick-brick-user-account-ui-component/manifests/styles.config.js [hunk 4]
```diff
@@ -245,9 +288,50 @@ const mobile = [
...
+          },
+        ],
+      },
+      {
+        key: "button_2_login_screen_type_default",
+        type: "text_input",
+        initial_value: "",
+        label: "Login Screen Time Default",
+        tooltip:
+          "Add screen mapping login_{{ctx/countryCode}}in this format, fallback logic",
```
Comment (ranmeirman) [inline plugins/quick-brick-user-account-ui-component/manifests/styles.config.js:327]: same comment as above

File: plugins/quick-brick-user-account-ui-component/manifests/styles.config.js [hunk 5]
```diff
@@ -655,7 +740,49 @@ const tv = [
...
+          },
+        ],
       },
+      {
+        key: "button_1_login_screen_type_default",
+        type: "text_input",
+        initial_value: "",
+        label: "Login Screen Time Default",
+        tooltip:
+          "Add screen mapping login_{{ctx/countryCode}}in this format, fallback logic",
```
Comment (ranmeirman) [inline plugins/quick-brick-user-account-ui-component/manifests/styles.config.js:776]: same comment as above

File: plugins/quick-brick-user-account-ui-component/manifests/styles.config.js [hunk 6]
```diff
@@ -842,9 +970,50 @@ const tv = [
...
+            key: "styles/button_1_login_type",
+          },
+        ],
+      },
+      {
+        key: "button_2_login_screen_type",
+        type: "text_input",
+        initial_value: "",
+        label: "Login Screen Time",
+        tooltip: "Add screen mapping login_{{ctx/countryCode}}in this format",
```
Comment (ranmeirman) [inline plugins/quick-brick-user-account-ui-component/manifests/styles.config.js:994]: same comment as above

File: plugins/quick-brick-user-account-ui-component/manifests/styles.config.js [hunk 7]
```diff
@@ -842,9 +970,50 @@ const tv = [
...
+          },
+        ],
+      },
+      {
+        key: "button_2_login_screen_type_default",
+        type: "text_input",
+        initial_value: "",
+        label: "Login Screen Time Default",
+        tooltip:
+          "Add screen mapping login_{{ctx/countryCode}}in this format, fallback logic",
```
Comment (ranmeirman) [inline plugins/quick-brick-user-account-ui-component/manifests/styles.config.js:1009]: same comment

**Review by ranmeirman — CHANGES_REQUESTED** [review]

**Review by inckie — APPROVED** [review]

**Review by ranmeirman — APPROVED** [review]

**Pull request merged**